### PR TITLE
quota: add quota package

### PR DESCRIPTION
**Pull request by christopherco**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

The quota package monitors and limits disk quota. Partners have requested this tool to be made available and supported in Mariner.
We supply this today in the extended repo, but this change moves it to our main SPECS folder so it will be published as part of our core package repo.

Signed-off-by: Chris Co <chrco@microsoft.com>

###### Change Log  

- Migrate the quota package from SPECS-EXTENDED to SPECS
- Update to quota v4.06 (imported from Fedora 36)

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- https://microsoft.visualstudio.com/OS/_workitems/edit/39419014
- https://microsoft.visualstudio.com/OS/_workitems/edit/38077011

###### Test Methodology

Local build
Pipeline build - aarch64(193428); x86_64(193415)

### CODE REVIEW

**Comment by christopherco** [comment]
Looks like there is a newer version of quota available (4.06). Converting back to draft to update to 4.06

File: SPECS/quota/quota.spec [hunk 1]
```diff
@@ -231,15 +206,6 @@ Linux/UNIX environment.
 %patch7 -p1
```
Comment (neha170) [inline SPECS/quota/quota.spec:206]: Nit: could use autosetup since there are many addition/ deletion from patchlist

**Review by neha170 — APPROVED** [review]

**Review by hbeberman — APPROVED** [review]

**Review by henryli001 — APPROVED** [review]

**Review by anphel31 — APPROVED** [review]

File: SPECS/quota/quota.spec (hunk 1 above)
Comment (christopherco) [reply SPECS/quota/quota.spec]: Thanks for the suggestion! Fixed!

Comment (christopherco) [reply SPECS/quota/quota.spec]: Thanks for the suggestion! Fixed!

**Pull request merged**